Solution1.java

```diff
@@ -0,0 +1,55 @@
+class Solution {
+    public int[] searchRange(int[] nums, int target) {
```

Comment:
Basically the logic for firstOccurence() and lastOccurence() is same. I would recommend you to use a flag and avoid duplication of code.

Reply:
okay